apps/system/app/[subdomain]/(dashboard)/escuela/[slug]/public/page.tsx

```
@@ -0,0 +1,379 @@
+import Header from "@/components/public/header";
+import SchoolMap from "@/components/public/mapa";
+import Footer from "@/components/public/footer";
+
+export default function EscuelaHome() {
+  const [currentSlide, setCurrentSlide] = useState(0); //estado ca
+  const { escuela } = useEscuela();
+  const setItems = useBreadcrumbStore((state) => state.setItems);
+
+  useEffect(() => {
+    if (escuela) {
+      setItems([{ label: `${(escuela?.nombre).toLocaleUpperCase()}` }]);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)